src/framework/services/filter/Filter.js

```diff
@@ -42,7 +42,9 @@ export default class Filter {
         return index !== -1;
       }
       case ComparisonTypes.ONE_OF: {
-        return this.comparisonValue.indexOf(itemValue) !== -1;
+        const comparisonTypeValue =
```

Comment:
`comparisonTypeValue` does not seem right. Better to use `normalizedComparisonValue` as in other cases.